Tests/Form/Widget/FormSonataFilterChoiceWidgetTest.php

```diff
@@ -64,7 +63,7 @@ public function testDefaultValueRendering()
 
     protected function getParentClass()
     {
-        if (version_compare(Kernel::VERSION, '2.8.0', '>=')) {
+        if (!method_exists('Symfony\Component\Form\FormTypeInterface', 'setDefaultOptions')) {
```

Comment:
This is not good enough, `setDefaultOptions` still exists on 2.8.

https://github.com/symfony/symfony/blob/2ddab1d055059e00a3477b09979123624db291fd/src/Symfony/Component/Form/FormTypeInterface.php#L80

You should reverse the condition.